src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java

```diff
@@ -211,10 +211,6 @@ public static void validateEndpoint(String endpoint) throws URISyntaxException,
         if (!VALID_ENDPOINT_COSMOS_INSTANCE_PATTERN.matcher(cosmosInstance).matches()) {
```

Comment:
do we need this thing as well? what's the pattern that we expect? What if the user specifies and IP address (ie see the local Cosmos DB emulator)?

Reply:
these validations are for the config UX to validate that users have entered valid values for various config settings. 
this is similar to the config checks on the portal when users supply a value for their endpoint, database name etc. 

the pattern is here - https://github.com/microsoft/kafka-connect-cosmosdb/blob/20cff56f8b03b0b51ab9df521182c8992351b080/src/main/java/com/azure/cosmos/kafka/connect/CosmosDBConfig.java#L30

[a-z0-9\\-]{3,44}